lib/utils/testLauncherHelper.js

```diff
@@ -145,12 +145,26 @@ function getDeviceLogStream(device, userArgs, startTime, logDir) {
 }
 
 /**
- * Verify the debug port parameter, can be port number, boolean or undefined
+ * Check if debug argument presents and return debug option
  * @param {Object} ownArgw - implicitly derived launched parameters
- * @returns {number|undefined|boolean}
+ * @returns {string | boolean}
  */
-function getDebugPort(ownArgw) {
-	return ownArgw.inspectBrk || ownArgw.inspect;
+function getDebugOption({inspectBrk, inspect}) {
+	const debuggerType = ramda.toPairs(ramda.pickBy(v => ramda.is(Number, v) || ['true', true].includes(v), {
+		'--inspect-brk': inspectBrk,
+		'--inspect': inspect,
+	}));
+
+	return debuggerType[0]
+		? `${debuggerType[0][0]}${ramda.is(Number, debuggerType[0][1]) ? '=' + debuggerType[0][1] : ''}`
+		: false;
+}
+
+function isDebugMode({inspectBrk, inspect}) {
```

Comment:
Would be cleaner if isDebugMode returned boolean and then you don't have to do !!isDebugMode